orchagent/neighorch.cpp

```diff
@@ -250,11 +259,11 @@ bool NeighOrch::setNextHopFlag(const NextHopKey &nexthop, const uint32_t nh_flag
     }
 
     nhop->second.nh_flags |= nh_flag;
-
+    uint32_t count;
```

Comment:
what is the purpose of this count here?

Reply:
This is the only place where valid/invalid nexthop group is used and the API is modified to return number of nexthop groups that were modified. This is a no-op here.  The caller can determine what to do with returned value